rasterio/rio/info.py

```diff
@@ -50,13 +55,22 @@
 @click.option('--checksum', 'meta_member', flag_value='checksum',
               help="Print integer checksum of a single band "
                    "(use --bidx).")
+@click.option('--gcps', 'meta_member', flag_value='gcps',
+              help="Print JSON representation of ground control points.")
```

Comment:
@perrygeo this occurred to me, too. Splitting JSON/GeoJSON dumping (with optional sequences and RS) out into a rio-gcp would certainly simplify rio-info. I'm not sure writing GCPs is worth doing right now: lots of code and tests and not useful except with TIFFs (that are mostly already georeferenced).